food-truck-frontend/src/components/CreateRouteRow.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {FormControlLabel, Input, Switch, TextField} from "@material-ui/core";
+
+type CreateRowState = {
+    days: string[],
+    routeName: string,
+    active: string
+}
+
+type CreateRowProps = {
+    truckId: string,
+}
+
+class CreateRouteRow extends React.Component<CreateRowProps, CreateRowState>{
+
+    constructor(props: CreateRowProps) {
+        super(props);
+        this.state = {
+            days: [],
+            routeName: "",
+            active: "Y"
+        }
+    }
+
+    render() {
+        return (
+            <div>
```

Comment:
Not a big deal but if you need a wrapper element React lets you do this with `<></>`